Rakefile

```diff
@@ -44,6 +44,58 @@ namespace :version do
 
 end
 
+class OverrideReader
```

Comment:
I think this would all be better placed in omnibus or somewhere more centralized unless we thnk chefdk is the only consumer. If it is, I'd putt this somewhere other than the Rakefile